crates/llvm-builder/tests/build.rs

```diff
@@ -92,6 +102,16 @@ fn debug_build_with_tests_coverage() -> anyhow::Result<()> {
         .assert()
         .success();
 
+    Command::cargo_bin(common::REVIVE_LLVM)?
+        .current_dir(test_dir.path())
+        .arg("build")
+        .arg("--llvm-projects")
+        .arg("clang")
+        .arg("--llvm-projects")
+        .arg("lld")
+        .assert()
+        .success();
+
```

Comment:
Shouldn't this and the below command be merged? This here builds LLVM, clang and LLD. Below builds LLVM with tests and coverage.